app/models/document.rb

```diff
@@ -41,6 +43,7 @@ class Document < ApplicationRecord
   validate :file_attached
   validate :numbered
   validate :invalidated_comment_present?
+  validate :created_date_is_in_the_past
```

Comment:
you can use numericality here (a date is an amount of seconds): https://stackoverflow.com/a/49063513

Reply:
keeping in record following pairing - we're not using numericality as it requires the data value to be an integer for the time comparison so it didn't do exactly what we were after